actions/websocket_actions.jsx

```diff
@@ -677,6 +678,10 @@ export function handleUserRemovedEvent(msg) {
             if (msg.data.remover_id === msg.broadcast.user_id) {
                 browserHistory.push(getCurrentRelativeTeamUrl(state));
             } else {
+                if (!getUser(state, msg.data.remover_id)) {
+                    await dispatch(loadUser(msg.data.remover_id));
+                }
+
                 const user = getUser(state, msg.data.remover_id) || {};
```

Comment:
Not a big deal but we could probably save a call to `getUser` if we restructure as such:
```js
let user = getUser(state, msg.data.remover_id);
                
if (!user) {
    await dispatch(loadUser(msg.data.remover_id));
    user = getUser(state, msg.data.remover_id) || {};
}
```

Reply:
yes, good point